### ISSUE TITLE
'Select Item' prompt isn't shown

### ISSUE THREAD

**Issue by jmcmichael** [ng-select/ng-select#747]
### Expected behaviour
If a field value is undefined or null, I would expect the selector to show the greyed-out 'Select Item' prompt displayed in the demos.

### Actual behaviour
I just see a blank selector - no prompt. I've tried prepending an object to the items array with a value of null and a name of 'Select Item' as a workaround, but the selector doesn't show that item when initially rendered, either.

### More Info
I am using [ngrx-forms](https://github.com/MrWolfZ/ngrx-forms) to hook the form state into the application state managed by ngrx, and providing them to the selector using the async pipe. I think ng-selector should be shielded from the ngrx-forms implementation details but I thought I'd mention that just in case (ng-selector appears to implement `ControlValueAccessor` properly, and updates the state w/o issue - thanks!).

ng-select version: 2.3.6

browser: latest versions of Chrome, Firefox

reproducible in demo page: I haven't tried yet but I will put together a demo tonight if I can't solve this.

**Comment by jmcmichael** [comment]
It may be a CSS issue. I am using the Clarity UI component library for everything except for complex select boxes, as theirs hasn't been released yet. I notice that the arrow icon is off center and some of the other styles look to be a bit off:

![screen shot 2018-08-14 at 9 13 33 am](https://user-images.githubusercontent.com/132909/44097194-d7c0f176-9fa2-11e8-8ce6-f436ae33e396.png)

I'm intending on writing a Clarity theme once I have the form working the way I want - but in the meantime, do you have any suggestions on if it's possible to completely isolate the ng-select component's CSS from Clarity's? I'm aware that Angular has the ability to isolate component styles, but am not sure how that would work with imported styles and components.

**Comment by jmcmichael** [comment]
OK, well this is embarrassing - I saw the very first reactive form demo and started hacking, assuming that w/o a placeholder attribute the selector would show a default 'Select Item', and didn't specify a placeholder in my first selector. Added the placeholder attribute, and it appeared.

Closing the issue (although you might want to consider adding a placeholder attribute to that first Reactive forms example).
